firestore-service.js

```diff
@@ -50,14 +50,19 @@ function initializeFirestore(keys) {
 
 async function getData({ pathname, query }) {
   try {
-    const { id, filters } = query;
+    const { id, limit } = query;
     let data = firestore.collection(pathname);
     data = await (id
       ? data
           .doc(id)
           .get()
           .then(item => ({ id: item.id, ...item.data() }))
-      : data.get().then(snapshot => snapshot.docs.map(doc => ({ ...doc.data(), id: doc.id }))));
+      : limit
+        ? data
+            .limit(Number(limit))
```

Comment:
isn't there a way to pass an argument to limit so that it gets all the available results, like `.limit(limit || 'infinite')` so that all that code isn't repeated? I did some research but couldn't find anything..

Reply:
I looked for that too, but also couldn't find anything. I think the main reason there is not a 'infinite' for limit is that if you don't want a limit you simply don't use `limit` on the query

Reply:
maybe this?
```
data.limit( limit ? Number(limit) : Infinity )
```